src/view.tsx

```diff
@@ -30,6 +49,7 @@ function ColorPicker(props: any) {
     <>
       <button
         className="jp-theme-editor-button-color"
+        type="button"
```

Comment:
I think I had to add this because then when clicking on any button on the theme editor even those of the colors pickers, the form was submitted and the post command was sent.